src/Integration.Vsix/SLCore/SLCoreEmbeddedPluginJarLocator.cs

```diff
@@ -33,16 +34,24 @@ namespace SonarLint.VisualStudio.Integration.Vsix.SLCore;
 public class SLCoreEmbeddedPluginJarLocator : ISLCoreEmbeddedPluginJarLocator
 {
     private const string JarFolderName = "DownloadedJars";
+
+    private readonly Dictionary<string, string> connectedModePluginNamePatternByPluginKey = new()
+    {
+        { "text", "sonar-text-plugin-(\\d+\\.\\d+\\.\\d+\\.\\d+)\\.jar" },
+        { "javascript", "sonar-javascript-plugin-(\\d+\\.\\d+\\.\\d+\\.\\d+)\\.jar" }
```

Comment:
Why include js? Didn't we agree we want to use the one from the server?

Reply:
Because the analysis is not working for typescript/javascript neither. This time I can see that the javascript embedded plugin from SlCore is used, but it doesn't seem to work at all. Not sure why

Reply:
Is it not working for SQ9.9 only? Would be nice to know the reason as well, because now we're forcing a worse UX for all users while not knowing exactly if it's the same problem or not

Reply:
For SQ 10.x it is working. Why is it worse UX ?